File: src/main/resources/output/IFC4X3_python/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcGroup/IfcAsset/IfcAsset.py
class IfcAsset(IfcGroup):
    def __init__(self):
        self.Identification: IfcIdentifier = None
        self.OriginalValue: IfcCostValue = None
        self.CurrentValue: IfcCostValue = None
        self.TotalReplacementCost: IfcCostValue = None
        self.Owner: IfcActorSelect = None
        self.User: IfcActorSelect = None
        self.ResponsiblePerson: IfcPerson = None
        self.IncorporationDate: IfcDate = None
        self.DepreciatedValue: IfcCostValue = None

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcGroup/IfcAsset/IfcAsset.cs
public class IfcAsset : IfcGroup
{
    public IfcIdentifier Identification { get; set; }
    public IfcCostValue OriginalValue { get; set; }
    public IfcCostValue CurrentValue { get; set; }
    public IfcCostValue TotalReplacementCost { get; set; }
    public IfcActorSelect Owner { get; set; }
    public IfcActorSelect User { get; set; }
    public IfcPerson ResponsiblePerson { get; set; }
    public IfcDate IncorporationDate { get; set; }
    public IfcCostValue DepreciatedValue { get; set; }
}
